integration-tests/pom.xml

```diff
@@ -102,12 +102,24 @@
             <version>2.4.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mongodb.spark</groupId>
+            <artifactId>mongo-spark-connector_${scala.binary.version}</artifactId>
+            <version>2.4.1</version>
+            <scope>test</scope>
+        </dependency>
         <dependency>
             <groupId>org.cassandraunit</groupId>
             <artifactId>cassandra-unit</artifactId>
             <version>2.1.9.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>de.flapdoodle.embed</groupId>
+            <artifactId>de.flapdoodle.embed.mongo</artifactId>
+            <version>1.47.1</version>
```

Comment:
There is no sense in adding `<version>` here if you have it in the `<dependencyManagement>` section below.

Otherwise LGTM.

Reply:
I'm not sure I am following. I think this is the only spot that the flapdoodle.embed.mongo is referenced? Should I move the version reference to the project level pom.xml?

Reply:
Oh, it was for the `mongo-spark-connector_${scala.binary.version}`, line #108. Sorry

Reply:
Ah got ya. I'll update that and the cassandra connector which has the same issue